packages/dds/tree/src/domains/schemaBuilder.ts

```diff
@@ -273,27 +273,27 @@ export class SchemaBuilder<
 	/**
 	 * {@link leaf.number}
 	 */
-	public readonly number = leaf.number;
+	public readonly number: typeof leaf.number = leaf.number;
```

Comment:
Some explicit types added where they were simple, to reduce the number of dynamic imports in `d.?ts` files.

Reply:
We shouldn't have to make any source changes to get rollups and generate API reports.
Alternate solution for tree #19141